client/storage/index.js

```diff
@@ -97,22 +108,27 @@ const getChunk = async (chunkId, encoding = 'utf8', useCache = true) => {
         const query = getDownloadQuery(chunkId);
 
         const queryResult = await request(config.get('storage.arweave_gateway_url'), query);
-        logger.debug({chunkId}, 'Graphql request success');
+        log.debug({chunkId}, 'Graphql request success');
+
         for (const edge of queryResult.transactions.edges) {
             const txid = edge.node.id;
             log.debug({chunkId, txid}, 'Downloading data from arweave');
 
             let data;
             let buf;
-            if (process.env.MODE === 'zappdev'){
-                data = (await axios.get('http://' + config.arweave_host + ':' + config.arweave_port + '/tx/' +  txid + '/data')).data;
+            
+            //TODO: Remove the if part (maintain else) when this bug of arlocal was fixed
+            //https://github.com/textury/arlocal/issues/63
+            if (config.get('storage.use_arlocal')){
+                data = (await axios.get('http://' +  config.get('storage.arweave_host') + 
+                    ':' + config.get('storage.arweave_port') + '/tx/' +  txid + '/data')).data;
```

Comment:
@diogosmendonca - note a contributor of Arlocal has posted a work around in the issue thread. I think the proposed work around will make our solution more complex and we should probably keep as we have for now. Lets discuss.